docs/elements/hx-drop-zone/index.html

```diff
@@ -28,7 +28,7 @@
           <ul class="hxList">
             <li>
               <code>drop</code> - See MDN's
-              <a href="https://goo.gl/aKLYsQ" target="_blank">
+              <a href="https://goo.gl/aKLYsQ" target="_blank" ref="noopener nofererer">
```

Comment:
TODO: `s/referer/referrer/`

Reply:
Case and point for #288.
Even I can't spell `noreferrer` correctly (or even misspell it _consistently_).